ocpmodels/datasets/single_point_lmdb.py

```diff
@@ -28,12 +30,13 @@ class SinglePointLmdbDataset(Dataset):
     def __init__(self, config, transform=None):
         super(SinglePointLmdbDataset, self).__init__()
 
-        self.config = config
+        self.db_path = Path(self.config["src"])
```

Comment:
`self.config` hasn't been defined. Hits an error.

Reply:
Oops, that line shouldn't have been removed. Fixed in a6030eb.